src/applications/claims-status/components/ClaimsListItem.jsx

```diff
@@ -3,28 +3,50 @@ import moment from 'moment';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router';
 
-import {
-  getPhaseDescription,
-  isClaimComplete,
-  getClaimType,
-} from '../utils/helpers';
+const statusMap = {
+  CLAIM_RECEIVED: 'Claim received',
+  INITIAL_REVIEW: 'Initial review',
+  EVIDENCE_GATHERING_REVIEW_DECISION:
+    'Evidence gathering, review, and decision',
+  PREPARATION_FOR_NOTIFICATION: 'Preparation for notification',
+  COMPLETE: 'Closed',
+};
+
+function getStatusDescription(status) {
+  return statusMap[status];
```

Comment:
I'm not sure what a sensible default here would be. The current code doesn't have a default case and if the status is null then the `Status:` field 
![Screenshot 2023-02-01 at 1 52 28 PM](https://user-images.githubusercontent.com/13838621/216148911-d46f9432-ec2f-436c-842c-b34507268eec.png)
ends up being empty